src/resources/candlepin-internationalization.sh

```diff
@@ -29,7 +29,7 @@ for GIT_BRANCH in main candlepin-4.1-HOTFIX candlepin-4.2-HOTFIX; do
   git checkout $GIT_BRANCH
   evalrc $? "Checkout branch : $GIT_BRANCH was not successful."
 
-  JAVA_VERSION=11
+  JAVA_VERSION=17
```

Comment:
we cannot just set java 17 for all branches. We want java 17 to be set only for the main branch, and Java 11 for older branches

Reply:
You can see here the code I removed, was doing this switch between 8 and 11: https://github.com/candlepin/candlepin-jobs/commit/013958b23ef3f014f4bc263503b70e0ce5d96955

Reply:
done